helpers/lbryApi.js

```diff
@@ -1,15 +1,27 @@
 var axios = require('axios');
+var db = require("../models");
 
 module.exports = {
-	publishClaim: function(publishParams){
+	publishClaim: function(publishParams, fileType){
 		var deferred = new Promise(function(resolve, reject){
 			console.log(">> lbryApi >> publishClaim:", publishParams);
 			axios.post('http://localhost:5279/lbryapi', {
 				"method": "publish", 
 				"params": publishParams
 			}).then(function (response) {
-				console.log(">> 'publish' success");
-				resolve(response.data);
+				console.log(">> 'publish' success", response);
+				db.File.create({
+					name: publishParams.name,
+					claim_id: response.data.result.claim_id,
+					outpoint: response.data.result.outpoint,
+					file_name: "test",
+					file_path: publishParams.filePath,
+					file_type: fileType,
+					nsfw: nsfw,
+				}).catch(function(error){
+					console.log('An error occurred when writing to the MySQL database:', error);
```

Comment:
agreed.  will wait and address this with the next big update which will be focused on logging.